router/server.go

```diff
@@ -102,6 +103,46 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	httpPorts := []int{*httpPort}
+	httpsPorts := []int{*httpsPort}
+	if portRaw := os.Getenv("DEFAULT_HTTP_PORT"); portRaw != "" {
+
+		if port, err := strconv.Atoi(portRaw); err != nil {
+			shutdown.Fatalf("Invalid DEFAULT_HTTP_PORTS: %s", err)
+		} else if port == 0 {
+			log.Warn("Disabling HTTP acccess (DEFAULT_HTTPS_PORT=0)")
```

Comment:
It should not nullify httpPorts, because someone might remove default HTTP access, yet still allow per-app routed HTTP access. You are right about the log message, however.